src/store/messages/saga.ts

```diff
@@ -477,9 +461,15 @@ export function* getPreview(message) {
   return yield call(getLinkPreviews, link[0].href);
 }
 
-function* syncChannelsTask(action) {
-  while (yield select(rawShouldSyncChannels(action.payload.channelId))) {
-    yield call(fetchNewMessages, action);
+function* pollForPublicChannelMessages() {
+  while (true) {
+    const currentUser = yield select(currentUserSelector());
+    const isPublicZOS = !currentUser?.id;
```

Comment:
Yeah, public ZOS is when you're not logged in. You can actually see public channels, etc, but you're not a user so you can't do things like...send messages, etc. It also means you can't get real time message events because we don't know who you are.